Issue # 2958 (#4559)
concourse/concourse: release-notes/latest.md

#### <sub><sup><a name="4507" href="#4507">:link:</a></sup></sub> fix

* @iamjarvo fixed a [bug](444://github.com/concourse/concourse/issues/4472) where `fly builds` would show the wrong duration for cancelled builds #4507.

#### <sub><sup><a name="2958" href="#2958">:link:</a></sup></sub> feature

> * @ctorboh15 added support for a CONCOURSE_TLS_ONLY configuration that will remove all non TLS api members at startup

Reviewer: ```suggestion
* @ctorboh15 added support for a CONCOURSE_TLS_ONLY configuration that will disable the non-TLS API listener.
```
